lib/internal/test_runner/mock/mock_timers.js

```diff
@@ -316,6 +317,10 @@ class MockTimers {
     this.#toggleEnableTimers(true);
   }
 
+  [SymbolDispose]() {
```

Comment:
Shouldn’t we do `MockTimers.prototype[SymbolDispose] = MockTimers.prototype.reset`? That’s how the ECMAScript spec usually does it.

Reply:
Why? 

(like, we don't guarantee function equality between .reset and Symbol.dispose and I don't find it more or less readable)

Reply:
You save memory by re-using the same function, and IMO the intent (that it's an alias) is clearer. Mimicking spec behavior sounds like the right thing to do (because that's what our users are used to). Not a blocking comment btw.

Reply:
Exhibit A: https://tc39.es/proposal-explicit-resource-management/#sec-disposablestack.prototype-@@dispose

I haven't done the work of digging through the TC39 meeting notes to see what are their justifications for doing this, but shouldn't the burden of proof be upon you who is doing things differently from the spec? Why did you choose to write it this way? I hope my wording doesn't read as argumentative, I'm genuinely curious 😊

Reply:
> but shouldn't the burden of proof be upon you who is doing things differently from the spec?

The spec defines the symbol and how `using` and `await using` calls it, it does not make any requirements on callers with regards to function equality especially with stuff like `reset` where it's our own internal API as far as I understand.

> Why did you choose to write it this way?

Just the mental model "disposing the mock timers rests them" rather than "disposing and resetting are necessarily the same thing" but that's just my mental model and if you feel strongly about it I'm happy to change it I just don't think it's more readable.

>  I hope my wording doesn't read as argumentative

As usual - reviews are valuable and help us keep a clean code base, constructive disagreements/debates are valuable and your comment does not come off as argumentative in the slightest to me (but thanks for checking that's also appreciated).

Reply:
> Just the mental model "disposing the mock timers rests them" rather than "disposing and resetting are necessarily the same thing" but that's just my mental model

Then maybe we shouldn’t document it as an alias? I don’t feel strongly, but I feel the current implementation and the current documentation are saying different things, which is not great.

Reply:
Ah, I see. I'll change the docs then 👍